### ISSUE TITLE
xmitgcm 0.5.1 changed the name of the variable's coordinates

### ISSUE THREAD

**Issue by liu-ran** [xgcm/xmitgcm#268]
Hi, today I updated the xmitgcm to the version (0.5.1). I found that the variable coordinates no longer have the 'XC', 'YC', and so on. As the picture shows, the variable 'U' uses the name 'i_g', 'j'. But these name are not supported by the xgcm package to do some differential calculations, such as vorticity and gradient. What can I do about this issue?  Many thanks.

![微信图片_20210528154734](https://user-images.githubusercontent.com/48266323/119949602-9ccf5880-bfcc-11eb-840f-3da286a0702a.png)

**Comment by AaronDavidSchneider** [comment]
Hi, this is actually a bug and will be fixed in #267.

**Comment by AaronDavidSchneider** [comment]
I forgot to mention. You can temporary fix the issue by downgrading `xarrray` to version `0.16.0`.
If you use anaconda:
```
conda install xarray==0.16
```

**Comment by AaronDavidSchneider** [comment]
> @AaronDavidSchneider Thanks. I find that if I use the xmitgcm to read one file ( single time record), the variable coordinates are right. But if I read many files with large time records, the variable coordinates would be wrong as the pictures show. I tried the xarray 0.16, but still can't get the right variable coordinates.

Yes that is exactly where the issue is: `xarray` changed the way datasets are combined in 0.18. This is why `xmitgcm` in combination with `xarray` 0.18 does not convert dimensions if multiple snapshots are loaded.

**Comment by rabernat** [comment]
Thanks for reporting this @liu-ran and sorry for the problems! It was caused by upstream changes in Xarray which we are indeed trying to fix in #267.

**Comment by daanreijnders** [comment]
I had the same issue and used the build version, since #267 has been merged. Thanks for fixing this! Are you planning a new release soon? Would be useful to people freshly installing xmitgcm from conda-forge.

**Comment by rabernat** [comment]
I just released 0.5.2. It should be on pypi very soon and conda-forge not long after.
